From 4cab0e38220d70246f4dde3ed72d95d4279df1ce Mon Sep 17 00:00:00 2001
From: Fugang Duan <b38611@freescale.com>
Date: Mon, 26 Oct 2015 14:07:55 +0800
Subject: [PATCH 0609/5242] MLK-11758 dts: mx6ul evk: set regulator wlreg_on
 always on

commit  ec66ddf6ec02c595f0d9a4834a530f14fcf7b5db from
https://source.codeaurora.org/external/imx/linux-imx.git

Murata adapter VDDIO is derived from sd slot VMMC supply, if usdhc cannot
detect sd/sdio card, VMMC is shutdown that causes VDDIO has no voltage, and
bt cannot work. So BT depends on VDDIO/VMMC.

Now the workaround is set wlreg_on alwasys on, usdhc can detect wifi card
and supply VMMC 3.3V voltage, so that Murata adapter VDDIO has power supply.

Signed-off-by: Fugang Duan <B38611@freescale.com>
(cherry picked from commit f17b5e5be37875e05794621eacf386ec25adf223)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi b/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
index 51e8bbd..1f3f8d5 100644
--- a/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
+++ b/arch/arm/boot/dts/imx6ul-evk-btwifi.dtsi
@@ -21,6 +21,7 @@
 			regulator-name = "wlreg_on";
 			gpio = <&gpio5 1 0>;
 			startup-delay-us = <100>;
+			regulator-always-on;
 			enable-active-high;
 		};
 	};
-- 
1.7.9.5

